pkg/agent/openflow/pipeline.go

```diff
@@ -476,6 +514,7 @@ func (c *client) l3FwdFlowToRemote(
 		Action().LoadRegRange(int(marksReg), portFoundMark, ofPortMarkRange).
 		// Flow based tunnel. Set tunnel destination.
 		Action().SetTunnelDst(tunnelPeer).
+		Action().LoadRegRange(int(tunnelDstReg), binary.LittleEndian.Uint32(tunnelPeer.To4()), binding.Range{0, 31}). // Traceflow
```

Comment:
If you consume match field "NXM_NX_TUN_IPV4_DST" in PacketIn, this line will not be needed in future.

Reply:
Fixed